src/main/java/com/sparta/popupstore/domain/review/controller/ReviewController.java

```diff
@@ -59,4 +66,15 @@ public ResponseEntity<Void> deleteReview(@AuthUser User user ,
         reviewService.deleteReview(user, reviewId);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+
+    @Operation(summary = "리뷰조회")
+    @Parameter(name = "id", description = "팝업스토어 고유번호")
+    @Parameter(name = "contents", description = "수정된 리뷰내용")
+    @Parameter(name = "star", description = "수정된 별점")
+    @Parameter(name = "name", description = "유저이름")
+    @GetMapping("/popupstores/{popupStoreId}")
+    public Page<ReviewFindAllResponseDto> findReviews(@AuthUser User user, @PathVariable Long popupStoreId,
```

Comment:
```suggestion
    public Page<ReviewFindAllResponseDto> findReviews(@PathVariable Long popupStoreId,
```
로그인 없이도 조회는 가능해야 할 것 같습니다.
이후에도 사용하지 않으면 user는 모두 빼주세요.